internal/config/identity/ldap/config.go

```diff
@@ -586,7 +586,7 @@ func Lookup(kvs config.KVS, rootCAs *x509.CertPool) (l Config, err error) {
 	l.ServerAddr = ldapServer
 	l.stsExpiryDuration = defaultLDAPExpiry
 	if v := env.Get(EnvSTSExpiry, kvs.Get(STSExpiry)); v != "" {
-		logger.Info("DEPRECATION WARNING: Support for configuring the default LDAP credentials expiry duration will be removed in a future release. Please use the `DurationSeconds` parameter in the LDAP STS API instead.")
```

Comment:
For username format mode, the deprecation notice needs to be near line 650 or so. This notice is for the `STSExpiry` setting, which we deprecated recently. We have not so far indicated official deprecation for username format - and there are some users, so a longer timeline is probably (2-3 months?) best.

Reply:
Nah just keep it 2 months @donatello there is no point in perpetuating username format

Reply:
let's just remove them together